docs/tasks/configure-pod-container/share-process-namespace.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Share Process Namespace between Containers in a Pod
+min-kubernetes-server-version: v1.10
+approvers:
+- dawnchen
+- verb
+---
+
+{% capture overview %}
+
+{% include feature-state-alpha.md %}
+
+This page shows how to configure process namespace sharing for a pod. When
+process namespace sharing is enabled, processes in a container will be visible
+to all other containers in that pod.
+
+This can be useful for cooperating containers, such as a log handler sidecar
+container, or troubleshooting container images that don't include debugging
+utilities like a shell.
+
+{% endcapture %}
+
+{% capture prerequisites %}
+
+{% include task-tutorial-prereqs.md %}
+
+A special **alpha** feature gate `PodShareProcessNamespace` has to be set to
+true across the system: `--feature-gates=PodShareProcessNamespace=true`.
+
+{% endcapture %}
+
+{% capture steps %}
+
+## Configure a Pod
+
+Process Namespace Sharing is enabled using the `ShareProcessNamespace` field of
+`v1.PodSpec`. For example:
+
+{% include code.html language="yaml" file="share-process-namespace.yaml" ghlink="/docs/tasks/configure-pod-container/share-process-namespace.yaml" %}
+
+1. Create the pod `nginx` on your cluster:
+
+        $ kubectl create -f https://k8s.io/docs/tasks/configure-pod-container/share-process-namespace.yaml
+
+1. Attach to the `shell` container and run `ps`:
+
+        $ kc attach -it nginx -c shell
+        If you don't see a command prompt, try pressing enter.
+        / # ps ax
+        PID   USER     TIME  COMMAND
+            1 root      0:00 /pause
+            8 root      0:00 nginx: master process nginx -g daemon off;
+           14 101       0:00 nginx: worker process
+           15 root      0:00 sh
+           21 root      0:00 ps ax
+
+1. It's possible to signal processes in other containers. Sending `SIGHUP` to
+   nginx causes it to restart the worker process (this requires the `SYS_PTRACE`
+   capability):
```

Comment:
Remove numbering -- the procedure is over. Also:
"You can signal processes ..."
and 
"Send `SIGHUP` to nginx to restart the worker process. This also requires the `SYS_PTRACE` capability."
(don't need colon)
Maybe add "For example," at the start of the second sentence, too?

Would providing the exact command here be useful?

Reply:
> Remove numbering -- the procedure is over. 

Yeah, I was a little confused about to handle this sort of thing since the entire thing isn't really a procedure. It's more of an arbitrary example. I removed the numbering where you suggested.

> Also:
> "You can signal processes ..."
> and
> "Send SIGHUP to nginx to restart the worker process. This also requires the SYS_PTRACE capability."
> (don't need colon)
> Maybe add "For example," at the start of the second sentence, too?

Done

> Would providing the exact command here be useful?

It's included. (It's `kill -HUP 8`)